src/services/organizationService.ts

```diff
@@ -63,7 +63,8 @@ export async function findAllNotPreDraft(
     `
       SELECT *
       FROM "${IpaPublicAdministrationModel.tableName}"
-      WHERE _search @@ plainto_tsquery('italian', :query);
+      WHERE _search @@ plainto_tsquery('italian', :query)
+      LIMIT ${Number(process.env.ADMINISTRATION_SEARCH_RESULTS_LIMIT) || 30};
```

Comment:
this setting and its default value must be in config.ts  -> https://github.com/teamdigitale/io-onboarding-pa-api/blob/master/src/config.ts